plugins/autodump.cpp

```diff
@@ -64,24 +57,32 @@ DFhackCExport command_result plugin_init ( color_ostream &out, vector <PluginCom
     return CR_OK;
 }
 
-DFhackCExport command_result plugin_shutdown ( color_ostream &out )
-{
+DFhackCExport command_result plugin_shutdown(color_ostream &out) {
     return CR_OK;
 }
 
-typedef map <DFCoord, uint32_t> coordmap;
+typedef map<DFCoord, uint32_t> coordmap;
 
-static command_result autodump_main(color_ostream &out, vector <string> & parameters)
-{
-    // Command line options
+// Projectile flags for falling items.
+static constexpr uint32_t proj_flags = (
+    df::projectile_flags::mask_no_impact_destroy |
+    df::projectile_flags::mask_bouncing |
+    df::projectile_flags::mask_piercing |
+    df::projectile_flags::mask_parabolic |
+    df::projectile_flags::mask_no_adv_pause |
+    df::projectile_flags::mask_no_collide
+);
```

Comment:
similar question to what I wrote in the gui/autodump PR - -can this logic be moved into `Items::makeProjectile`?

Reply:
Might not be desirable because these are the flags for falling items, not arrows, siege ammo, etc.

BTW, is there any way to avoid `df::projectile_flags::` for each of these? I tried various `using` statements with no luck.

Reply:
the parent enum Mask might have something to do with it. I saw the same thing in `buildingplan.cpp`